scripts/deploy-frontend.sh

```diff
@@ -41,6 +41,7 @@ update_frontend()
     fi
 
     cf map-route "$CGHOSTNAME_FRONTEND" app.cloud.gov --hostname "${CGHOSTNAME_FRONTEND}"
+    cf map-route "$CGHOSTNAME_FRONTEND" tanfdata.acf.hhs.gov 
```

Comment:
For clarification, wouldn't we only want to run this after we check the `CF_SPACE` first? Or is this change only for testing at this stage?